### DISCUSSION TITLE
晚上突然被限速了。挂载的阿里，也已经更新到最新版本

### DISCUSSION THREAD

**Discussion by Yolo0123** [AlistGo/alist#5610] [Q&A]
![Snipaste_2023-11-29_21-53-21](https://github.com/alist-org/alist/assets/128347102/378a2515-36dd-4fe2-84c7-b5fbfb0d8f30)
怎么搞阿？？？？？
以后不能用了吗？

**Comment by 1984Li** [comment]
同限速，应该是阿里动手了。

**Reply by zhoutao-cn** [reply]
应该是接口问题，小白羊没有限速，说明还有救。

**Reply by tonsen911** [reply]
我是更奇怪，4K的限速在4M，非4K的能跑几十M

**Comment by shaoyang778899** [comment]
还偶尔掉呢

**Reply by shaoyang778899** [reply]
failed link: failed get link: UserNotAllowedAccessResource:user not allowed access resource for violation

**Reply by Deepseaon** [reply]
你报这个是账户被锁了吧，小心点别滥用

**Comment by zhoutao-cn** [comment]
是的，晚上突然限速了，而且是专门对 Alist 限速，用“小白羊”就是满速。

**Comment by Great-old-one** [comment]
我还以为是我的问题，折腾了一晚上都不行

**Comment by w8571528** [comment]
我也一样，infuse直连阿里速度正常，只有alist webdav挂载速度被限制在2m/s以下。

**Reply by kratos9394** [reply]
确实如此

**Comment by bruce-wei** [comment]
同被限速。。。

**Comment by KevinLi2596** [comment]
同被限速。。。我还以为是阿里云又搞限速，结果直接通过webdav拷贝没有限制，那就是alist的问题了。

**Reply by Truman67** [reply]
请问怎么用webdav直接拷贝？

**Comment by zh826256645** [comment]
同样被限速，双 11 刚买了 3 年，就整这出😔

**Reply by ypq123456789** [reply]
白买了😔

**Reply by Banye-Jeffrey** [reply]
会员不限

**Reply by gzgzcyq** [reply]
会员也限速？

**Comment by citylyz** [comment]
用网页界面的阿里云盘默认播放器aliyun video previewer，没有限速，是试看要定时刷新。如果选video模式，就是限速了，只有500KB，相当于用第三方播放器，例如kodi等。感觉是用非阿里云播放器限速了。

**Reply by zhoutao-cn** [reply]
小白羊还是满速的。估计是用 Alist 挂资源的太多了，专门针对性的限速。

**Reply by w8571528** [reply]
网页是转码的，画质稀烂

**Comment by kitkw** [comment]
等待修复

**Comment by kratos9394** [comment]
估计是玩不起，要收钱了吧，限制到500k

**Comment by pongfcnkl** [comment]
可以自己去阿里云盘开发者申请申请appId 和Secret  然后用自己的api申请token
阿里云盘开发者：https://www.aliyundrive.com/developer/f
调用的php程序：https://github.com/520yxl/Aliyundrive_Refresh_Token_php
另外需要一台服务器和域名拿来申请token

**Reply by zhoutao-cn** [reply]
亲，对于我们“伸手党”来说，这样的玩法太高端了。

**Reply by formingform** [reply]
开发者申请需要很多东西，甚至还要ICP备案号，估计难申请。另外，没有看到需要一台服务器和域名来申请token啊？

**Reply by pongfcnkl** [reply]
@formingform 
不需要备案号，没有红点的不是必填项，申请的话越早越容易申请
服务器和域名是拿来搭建申请站的，毕竟api不支持无服务器申请。
也可以不需要服务器和域名，下面作者提供了服务器，你只需要输入id和Secret就行了

**Reply by shaoyang778899** [reply]
你好,问下申请开发者需要怎么填写啊.填了2次都没过

**Reply by shaoyang778899** [reply]
对接阿里云盘 OpenAPI 的应用场景简述.这个需要怎么填写啊.

**Reply by YouKnowMyEggs** [reply]
感谢，今天下午申请到了阿里云开发者账号，立马测试了一下，1000M宽带下能跑到平均500mbps。

**Reply by Jzx12138c** [reply]
现在又不行了，开发者也限速了

**Comment by 520yxl** [comment]
> 可以自己去阿里云盘开发者申请申请appId 和Secret 然后用自己的api申请token 阿里云盘开发者：https://www.aliyundrive.com/developer/f 调用的php程序：https://github.com/520yxl/Aliyundrive_Refresh_Token_php 另外需要一台服务器和域名拿来申请token

**千万不要用开发者账号绑定的网盘账号进行自己授权自己的开发应用密钥上,有可能会封禁**

**Reply by ypq123456789** [reply]
大佬，我已经这样操作过了，还来得及补救吗？

**Reply by pongfcnkl** [reply]
让应用的使用者直接使用应用开发者的 token
这句话的意思简单来说就是 不要用自己的账号做视频站 文件站等各种站点 （包括别人的号）不是不让你登录自己的号授权。
 还有阿里和115一样，最多只能用10线程，超出也会被ban

**Reply by ypq123456789** [reply]
原来如此，所以还是不分享就可以对吧，我就授权自己的号自己一个人看，应该没事吧

**Reply by 520yxl** [reply]
暂时不知道,我是大号申请了开发者,小号做授权存储的,现在小号申请了开发者,还没给我开。
我有空时看看有没有其他方式

**Reply by pongfcnkl** [reply]
直接授权给大号，没问题的

**Reply by 520yxl** [reply]
大号直接授权作者的，小号授权给大号

**Reply by pongfcnkl** [reply]
作者的api限速，你还不如直接授权你自己的开发者号

**Reply by wangrui1573** [reply]
怎么操作，用https://github.com/520yxl/Aliyundrive_Refresh_Token_php 获取到token，直接替换alist的token吗？

**Reply by 520yxl** [reply]
首先你要有阿里云盘的开发者应用的appId 和Secret ,然后修改代码中的appId 和Secret字段,修改后上传到支持PHP环境的服务器进行外网访问,回调地址是你的网站+/aliyundrive.php

**Reply by Lang0722** [reply]
申请完refresh token之后，服务器需要一直运行这个php服务吗？

**Reply by wangrui1573** [reply]
有要求 https吗，我有个VPS，随便挂个端口 或者上verlcel，serverless 行不？

**Reply by 520yxl** [reply]
网址带端口,阿里云盘的开发者应用的回调地址也要加上端口

**Reply by pongfcnkl** [reply]
需要https

**Reply by wangrui1573** [reply]
收到，我用vercel试试

**Reply by 520yxl** [reply]
我代码中是自动获取网站,自动拼接回调地址的,建议只使用https://和http://

**Reply by wangrui1573** [reply]
顺便问下，申请阿里云开发者，流程要跑多久？

**Reply by 520yxl** [reply]
不知道喔,我小号前几天申请的,现在还没有收到通知

**Reply by Lang0722** [reply]
换了别的类似alist的软件去申请refresh token再挂载还是限速的，这是开启白名单了吧？

**Reply by wangrui1573** [reply]
不是，我也遇到你这个问题，直到我在alist中新建一个存储，而不是修改旧存储

**Reply by 520yxl** [reply]
我这边没有限速呀,挂载的是我小号的账号,用的是自己的开发者应用信息

**Reply by 520yxl** [reply]
![微信截图_20231201171207](https://github.com/alist-org/alist/assets/29514859/bcf211b7-6a5d-4259-abff-dfd1d3cfe634)

**Reply by wangrui1573** [reply]
收到通知了，耗时5天

**Reply by TimeController3** [reply]
大佬，请问一下你的这句话：千万不要用开发者账号绑定的网盘账号进行自己授权自己的开发应用密钥上,有可能会封禁
意思是不是申请自己的api最好是额外新弄一个阿里云盘账号，搞到api后再把api应用到自己真正要用的阿里云盘账号上？
就是说不要用阿里云盘主账号去申请开发者api？

**Reply by Jzx12138c** [reply]
现在又不行了，开发者也限速了，自己授权自己，用了好久都没事，6月限速了

**Reply by Deepseaon** [reply]
其实还好，应该不是真限速，具体什么原因搞不懂，，不过有时候突然会卡，速度只有2m以下，但是快的时候能跑4m以上，我这网的话最高跑到过8m，但是具体因为什么导致限速没搞懂，反正目前的话看视频码率我这平均就是3-5m吧

**Reply by Jzx12138c** [reply]
> 其实还好，应该不是真限速，具体什么原因搞不懂，，不过有时候突然会卡，速度只有2m以下，但是快的时候能跑4m以上，我这网的话最高跑到过8m，但是具体因为什么导致限速没搞懂，反正目前的话看视频码率我这平均就是3-5m吧

4M就是限速了，原盘、高码率视频看不了，80M码率视频要10MB/s才能不卡，以前多了不说三四十的速度是有的，限速4M后用开发者这一招可以不限速，现在又不行了

**Reply by Deepseaon** [reply]
要这么说的话，还真是，我可能是因为很少看蓝光所以感知不是很强，不过有时候直接用链接也能跑到8m左右，但用挂载的webdav就慢

**Reply by Deepseaon** [reply]
很奇怪，今天专门找了个视频试了试，我用平板vlc看蓝光视频速度能有13000-20000kbs，但是我电脑直接用potplayer webdav就几m/s
话说你用的是自己的开发者api吗，我申请了两次快一礼拜了也没任何消息。。。

**Reply by tegrason** [reply]
> 要这么说的话，还真是，我可能是因为很少看蓝光所以感知不是很强，不过有时候直接用链接也能跑到8m左右，但用挂载的webdav就慢

webdav用开发者API的情况下可以跑到7M，不用的话只能跑1M

**Reply by tegrason** [reply]
> 很奇怪，今天专门找了个视频试了试，我用平板vlc看蓝光视频速度能有13000-20000kbs，但是我电脑直接用potplayer webdav就几m/s 话说你用的是自己的开发者api吗，我申请了两次快一礼拜了也没任何消息。。。

重新申请就行了

**Reply by Jzx12138c** [reply]
> 很奇怪，今天专门找了个视频试了试，我用平板vlc看蓝光视频速度能有13000-20000kbs，但是我电脑直接用potplayer webdav就几m/s 话说你用的是自己的开发者api吗，我申请了两次快一礼拜了也没任何消息。。。

我自己的，申请没过没啥办法，现在也不用申请了，反正没用。b和B要区分开16000kbps也就2兆B每秒

**Reply by Jzx12138c** [reply]
> > 要这么说的话，还真是，我可能是因为很少看蓝光所以感知不是很强，不过有时候直接用链接也能跑到8m左右，但用挂载的webdav就慢
> 
> webdav用开发者API的情况下可以跑到7M，不用的话只能跑1M

我用开发者API也就4M，速度限制和以前不用一样，也许不同地区拉的闸不一样吧。

**Reply by Deepseaon** [reply]
申请个自己的应该还是有点用的，之前有几天好像不就是官方提供的api用的人多了，token死活刷不出来

**Reply by Deepseaon** [reply]
可能是吧，有些地区能好点，试了试70g的星际穿越，卡到是不卡，但是速度就是过山车，一会高一会低一会，高了能有90000kbs低了就10000kbs，会员也限速真是服了。。。。。

**Reply by Jzx12138c** [reply]
> 可能是吧，有些地区能好点，试了试70g的星际穿越，卡到是不卡，但是速度就是过山车，一会高一会低一会，高了能有90000kbs低了就10000kbs，会员也限速真是服了。。。。。

已经比我们非会员高一档了，我最高三万五左右，完全ppt。打算不玩webdav看原盘了，我百度云有SVIP，回归百度云，回归下载

**Reply by Deepseaon** [reply]
![image](https://github.com/alist-org/alist/assets/111950052/bb53ac2f-7537-4e11-beb8-3bdbb0da965e)
这就是极限了
不过你有度盘svip那肯定度盘好使啊，我也是因为度盘会员太贵了我才转的阿里云，毕竟百度不充会员现在是完全没法用，720p都是限免
用alist也就是图方便，玩bt的大盘鸡我也有不过bt的话就非得放电脑上看了，还是想组台nas

**Reply by Jzx12138c** [reply]
> ![image](https://private-user-images.githubusercontent.com/111950052/344097045-bb53ac2f-7537-4e11-beb8-3bdbb0da965e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1NjQzMjIsIm5iZiI6MTcxOTU2NDAyMiwicGF0aCI6Ii8xMTE5NTAwNTIvMzQ0MDk3MDQ1LWJiNTNhYzJmLTc1MzctNGUxMS1iZWI4LTNiZGJiMGRhOTY1ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyOFQwODQwMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZGJmZDRiZTJjMTVlZTgwNzk4ODg3ZWRhM2UyNzA3ZmU0ZTkzYjJmZDdjYTU0OTZmMzY4NjhkNTJhN2NlNTdmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IpBPMgzmkT72EL1XKqkv6U5AKfNw4QrKtaDA7CyCygU) 这就是极限了 不过你有度盘svip那肯定度盘好使啊，我也是因为度盘会员太贵了我才转的阿里云，毕竟百度不充会员现在是完全没法用，720p都是限免 用alist也就是图方便，玩bt的大盘鸡我也有不过bt的话就非得放电脑上看了，还是想组台nas

主要是百度网盘webdav播放不好用

**Reply by Deepseaon** [reply]
申请到开发者了，感觉确实有提升，现在webdav启动就是9m起步了，起码能用，再后面估计就要上他那第三方权益包了，现在能用多久用多久吧。
不过我要想换回官方api只要把token换回官方客户端生成的就行吧？

**Comment by MatthewTt** [comment]
是的，挂载的限速了

**Comment by xhofe** [comment]
现已支持使用自己的client，在此处获取：https://alist.nn.ci/tool/aliyundrive/request.html
回调地址为： https://alist.nn.ci/tool/aliyundrive/callback
添加存储时，将client_id和client_secret也填上即可。

**Reply by wangrui1573** [reply]
client_id和client_secret 需要先成为开发者才能获取吗？

**Reply by 520yxl** [reply]
是的

**Reply by WXZhao7** [reply]
整了个开发者，按图中标注的填写提示`{"code":"InvalidRedirectUri","message":"invalid redirect_uri"`。请教一下是哪里出问题啦
![image](https://github.com/alist-org/alist/assets/56864878/e6928f99-2f23-4853-ab79-81cc6cc32a09)

**Reply by 520yxl** [reply]
回调地址没有设置正确

**Reply by WXZhao7** [reply]
设置为这两个好像还是不对，有点不明白了
![image](https://github.com/alist-org/alist/assets/56864878/dd2e9449-322c-4875-9645-ef63338b691e)

**Reply by 520yxl** [reply]
https://api.nn.ci/alist/ali_open/token

**Reply by 520yxl** [reply]
试一下这个链接,不行的话,就自建一下https://github.com/520yxl/Aliyundrive_Refresh_Token_php

**Reply by xhofe** [reply]
回调地址是 https://alist.nn.ci/tool/aliyundrive/callback

**Reply by WXZhao7** [reply]
多谢大佬，正在尝试用azure建（php直接运行那个文件后indirect还是不对，不想再学习了）。经测试，回调地址填 https://alist.nn.ci/tool/aliyundrive/callback 完美工作: )。删除原来的存储并填写新的刷新令牌，客户端ID和客户端密钥后可以突破4m/s的限制了，效果如下
![image](https://github.com/alist-org/alist/assets/56864878/db4b2e17-dea4-454d-8168-18af35e0b66c)

**Reply by WXZhao7** [reply]
我是没有搭建服务器生成**刷新令牌**，那个令牌链接好像无所谓的。我这的提问是借大佬服务器生成refresh_token，你这刷新令牌和ID密钥都有应该正常工作的吧

**Reply by TimeController3** [reply]
你好，请问一下，https://alist.nn.ci/tool/aliyundrive/request.html这个链接里面的回调地址只有两个可选而且不能改，要怎么使用你所说的https://alist.nn.ci/tool/aliyundrive/callback这个回调地址呢？

**Reply by TimeController3** [reply]
老哥你是怎么申请的，需要备案啥的吗

**Reply by WXZhao7** [reply]
> 你好，请问一下，https://alist.nn.ci/tool/aliyundrive/request.html这个链接里面的回调地址只有两个可选而且不能改，要怎么使用你所说的https://alist.nn.ci/tool/aliyundrive/callback这个回调地址呢？

成为开发者后，在阿里云盘开发者界面填这个回调地址就能免服务器用自己的应用ID和密钥生成refresh_token了

**Reply by WXZhao7** [reply]
> 老哥你是怎么申请的，需要备案啥的吗

不需要，直接申请就完事

**Reply by TimeController3** [reply]
好的好的，谢谢老哥

**Reply by rianjskis** [reply]
> 我是没有搭建服务器生成**刷新令牌**，那个令牌链接好像无所谓的。我这的提问是借大佬服务器生成refresh_token，你这刷新令牌和ID密钥都有应该正常工作的吧

您好，在添加存储中填写生成的refresh_token，还需要把Oauth令牌链接修改成回调地址吗，客户端ID与客户端密钥需要填写开发者的id与密钥吗

**Reply by WXZhao7** [reply]
> > 我是没有搭建服务器生成**刷新令牌**，那个令牌链接好像无所谓的。我这的提问是借大佬服务器生成refresh_token，你这刷新令牌和ID密钥都有应该正常工作的吧
> 
> 您好，在添加存储中填写生成的refresh_token，还需要把Oauth令牌链接修改成回调地址吗，客户端ID与客户端密钥需要填写开发者的id与密钥吗

Oauth令牌链接我没动，改了refresh_token，客户端ID/客户端密钥

**Reply by rianjskis** [reply]
> > > 我是没有搭建服务器生成**刷新令牌**，那个令牌链接好像无所谓的。我这的提问是借大佬服务器生成refresh_token，你这刷新令牌和ID密钥都有应该正常工作的吧
> > 
> > 您好，在添加存储中填写生成的refresh_token，还需要把Oauth令牌链接修改成回调地址吗，客户端ID与客户端密钥需要填写开发者的id与密钥吗
> 
> Oauth令牌链接我没动，改了refresh_token，客户端ID/客户端密钥

感谢告知，

**Reply by rianjskis** [reply]
> > > 我是没有搭建服务器生成**刷新令牌**，那个令牌链接好像无所谓的。我这的提问是借大佬服务器生成refresh_token，你这刷新令牌和ID密钥都有应该正常工作的吧
> > 
> > 您好，在添加存储中填写生成的refresh_token，还需要把Oauth令牌链接修改成回调地址吗，客户端ID与客户端密钥需要填写开发者的id与密钥吗
> 
> Oauth令牌链接我没动，改了refresh_token，客户端ID/客户端密钥

这样还限速吗

**Reply by WXZhao7** [reply]
> > > > 我是没有搭建服务器生成**刷新令牌**，那个令牌链接好像无所谓的。我这的提问是借大佬服务器生成refresh_token，你这刷新令牌和ID密钥都有应该正常工作的吧
> > > 
> > > 您好，在添加存储中填写生成的refresh_token，还需要把Oauth令牌链接修改成回调地址吗，客户端ID与客户端密钥需要填写开发者的id与密钥吗
> > 
> > Oauth令牌链接我没动，改了refresh_token，客户端ID/客户端密钥
> 
> 这样还限速吗

不限速了

**Reply by oyal** [reply]
~~请教一下，我使用 AppID 和 App Secret 生成的 `refresh_token` 在添加存储保存时提示 `Failed init storage: failed to refresh token: invalid client_id`。这个是什么原因呢？~~
添加存储要把 `客户端ID` `客户端秘钥` 也带上

**Reply by Mixlining** [reply]
请问Oauth令牌链接需要更换成什么

**Reply by zhychen1173** [reply]
> 现已支持使用自己的client，在此处获取：https://alist.nn.ci/tool/aliyundrive/request.html 回调地址为： https://alist.nn.ci/tool/aliyundrive/callback 添加存储时，将client_id和client_secret也填上即可。

我用 https://alist.nn.ci/tool/aliyundrive/request.html 使用自己的client_id 和 client_secret ，然后Scan QRcode ，手机app 授权之后，点击 I have scan，跳转页面报错，报：InvalidClientId: invalid client_id
InvalidClientId: invalid client_id

请教应该如何解决？谢谢。

**Reply by tegrason** [reply]
> > 现已支持使用自己的client，在此处获取：https://alist.nn.ci/tool/aliyundrive/request.html 回调地址为： https://alist.nn.ci/tool/aliyundrive/callback 添加存储时，将client_id和client_secret也填上即可。
> 
> 我用 https://alist.nn.ci/tool/aliyundrive/request.html 使用自己的client_id 和 client_secret ，然后Scan QRcode ，手机app 授权之后，点击 I have scan，跳转页面报错，报：InvalidClientId: invalid client_id InvalidClientId: invalid client_id
> 
> 请教应该如何解决？谢谢。

我也是，解决了吗？

**Reply by kinrt** [reply]
> https://alist.nn.ci/tool/aliyundrive/callback

使用登录，不要扫码。

**Comment by elevenface** [comment]
目前看起来还是非常慢

**Comment by synthetise** [comment]
同限速，但是我很轻度使用啊，现在也是2mb/s。。。

**Comment by jueinin** [comment]
限速到500kb了  直接没法看剧了

**Comment by YoungLee-coder** [comment]
好像好点了，现在能跑到7mb/s

**Reply by bruce-wei** [reply]
是的，我的也是8mb/s，估计是限速值提高了

**Reply by Benjamin23333** [reply]
我的也恢复了一些，但是只有3.7-3.8左右，可能是上午尝试绕开限速的时候，用aria2下载忘了设置并发数小于10被阿里风控了？

**Reply by synthetise** [reply]
同恢复到9mb/s左右

**Comment by zhoutao-cn** [comment]
12月2日下午恢复到10M左右的速度。达不到满速，可以看FHD视频。

**Reply by formingform** [reply]
请问看4K视频，要多少速度？

**Reply by synthetise** [reply]
如果要仅仅是下载的流媒体版本的话，感觉极限点2mb/s也够流畅播放了

**Reply by zhoutao-cn** [reply]
18M左右。

**Comment by Nick-lizheng** [comment]
我还是3m/s.....

**Comment by JIeJaitt** [comment]
感觉阿里网盘朝令夕改的，太没有安全感了

**Comment by kratos9394** [comment]
确实又好了，还好放的都是些垃圾视频

**Comment by limu96966** [comment]
4MB/s

**Comment by WXZhao7** [comment]
稳定不能过4m/s

**Comment by angelghost44** [comment]
百兆速度大概在四十多G的文件可以流畅，4m/s大概可以播放10-20G左右的文件，以后存都存265的流媒体版本了，如果不弄自己的clientid的话，只能做到这种程度了

**Comment by 520yxl** [comment]
https://api.alistcn.top/free_token/
根据作者的OAuth验证api进行了自建的api，暂时支持自有阿里云盘开发者应用的client_id和client_secret

**Reply by wangrui1573** [reply]
这个可以开源下吗，我在alist调用回调，写你那个php好像不行。。
![image](https://github.com/alist-org/alist/assets/42831156/414e066e-ef2b-4ce0-a92f-fb3f3cf9889a)

**Reply by 520yxl** [reply]
**暂时支持自有阿里云盘开发者应用的client_id和client_secret**，没有申请通过阿里云盘开发者，请不要使用

**Comment by Great-old-one** [comment]
会员现在还限速吗？

**Reply by zhoutao-cn** [reply]
Alist 挂载都限速。

**Reply by sephiroth233** [reply]
不只是alist，我是用小白羊也限速，只要授权第三方，我用阿里云盘pc端都被限速了，取消所有授权就好了，离谱

**Reply by sephiroth233** [reply]
限速已恢复，不再是300k了

**Comment by LGuangming** [comment]
为什么我没有用自己app获取token，直接用作者的获取到的refrensh token挂载下载速度飙到100多M/S，甚至比之前没限速的时候还高

**Reply by zhoutao-cn** [reply]
XD，请问你是怎么操作的？

**Reply by LGuangming** [reply]
> XD，请问你是怎么操作的？

我什么都没操作 测试自己就这样 估计是bug

**Reply by zhoutao-cn** [reply]
“直接用作者的获取到的refrensh token”——这一句我没看懂，请详细讲讲吧。谢谢啊！

**Comment by devyujie** [comment]
还是限速

**Comment by oyal** [comment]
限速3.8M😭PotPlayer看视频卡顿

**Comment by RyanLu99** [comment]
![image](https://github.com/alist-org/alist/assets/53104358/1f751f5e-4a1a-4426-ad9a-34159bba85db)
的确限速了，播放的视频码率在3.8M左右，看不了4K原盘了😭

**Reply by mxp108** [reply]
你好，我想请问一下potplayer网络速度怎么查看啊，我找不到开启的选项
![pot](https://github.com/alist-org/alist/assets/61874118/94e5b484-313d-489b-82ce-b9c23ecd99c0)

**Reply by zhoutao-cn** [reply]
不同的皮肤网速显示的地方不一样，而且和视频编码也有关系，有的不显示。

**Comment by ypq123456789** [comment]
我默认就开的，我也不知道怎么开，你可以百度一下或者问问gpt。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年2月8日(周四) 晚上8:54
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alist-org/alist] 晚上突然被限速了。挂载的阿里，也已经更新到最新版本 (Discussion #5610)




 
你好，我想请问一下potplayer网络速度怎么查看啊，我找不到开启的选项
 pot.jpg (view on web)

**Reply by mxp108** [reply]
嗯嗯，后面发现了  直接potplayer里面添加webdav就可以显示了   但感觉换了token也没有多大提升

**Comment by WuYang09661** [comment]
使用过各种方法了 更换api 套用cf 还是不行 一样限速 只能到900kb 有别的办法吗

**Comment by myzhongqiang** [comment]
我限速了 可怕啊

**Comment by AspirantH** [comment]
如何获取Client id呢

**Reply by Deepseaon** [reply]
需要自己注册开发者
